src/wwwroot/directives/breadcrumb.js.orig

```diff
@@ -0,0 +1,47 @@
+(function () {
```

Comment:
Seems you unintentionally include this merge file

Reply:
good catch!